keras_contrib/layers/normalization.py

```diff
@@ -319,8 +319,8 @@ def call(self, inputs, training=None):
 
         # update r_max and d_max
         t_val = K.get_value(self.t)
```

Comment:
I believe the real problem is here. K.get_value() runs a TensorFlow session internally to read the value of self.t and returns a float value. This value is 0 when you create the model. And since the call() function is called only once, then t_val will always be zero, and the following calculations will always be based on a t_val value of 0